charts/prometheus/templates/cm.yaml

```
@@ -20,6 +20,10 @@ data:
 {{- if eq $key "prometheus.yml" }}
     global:
 {{ $root.Values.server.global | toYaml | trimSuffix "\n" | indent 6 }}
+{{- if $root.Values.server.external_labels }}
+      external_labels:
+{{ $root.Values.server.external_labels | toYaml | indent 8 }}
+{{- end }}
```

I'd suggest to name the field with camelCase as usual, i.e. `externalLabels`.

```suggestion
{{- with $root.Values.server.externalLabels }}
      external_labels: {{ toYaml . | nindent 8 }}
{{- end }}
```

---

Agreed